src/DotNetWorker.Grpc/Factory/Contracts/IGrpcWorkerMessageFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using MsgType = Microsoft.Azure.Functions.Worker.Grpc.Messages.StreamingMessage.ContentOneofCase;
+
+namespace Microsoft.Azure.Functions.Worker.Grpc.Factory.Contracts;
```

Comment:
I think root directory name can be changed from "Factory" to "MessageHandling".

Reply:
Agree, much better name.